music21/metadata/__init__.py

```diff
@@ -96,161 +184,499 @@ class Metadata(base.Music21Object):
+
+        >>> metadata.Metadata.isStandardKey('marcrel:LBT')
+        True
+        >>> metadata.Metadata.isStandardKey('librettist')
+        True
+        >>> metadata.Metadata.isStandardKey('not a standard property')
+        False
+        >>> metadata.Metadata.isStandardKey('dcterms:nonstandardname')
+        False
+        >>> metadata.Metadata.isStandardKey(None)
+        False
+        '''
+        if not key:
+            return False
+
+        if Metadata._isStandardNSKey(key):
+            return True
+
+        if Metadata._isStandardUniqueName(key):
+            return True
+
+        return False
+
+    @staticmethod
+    def nsKeyToUniqueName(nsKey: str) -> t.Optional[str]:
+        '''
+        Translates a standard property NSKey ('namespace:name') to that
+        standard property's uniqueName.
+        Returns None if no such standard property exists.
+
+        >>> metadata.Metadata.nsKeyToUniqueName('marcrel:LBT')
+        'librettist'
+        >>> metadata.Metadata.nsKeyToUniqueName('not a standard nskey') is None
+        True
+        >>> metadata.Metadata.nsKeyToUniqueName(None) is None
+        True
+        >>> metadata.Metadata.nsKeyToUniqueName('dcterms:alternative')
+        'alternativeTitle'
+        '''
+        if not nsKey:
+            return None
+        uniqueName: t.Optional[str] = properties.NSKEY_TO_UNIQUE_NAME.get(nsKey, None)
+        return uniqueName
+
+
+# -----------------------------------------------------------------------------
+#   Pre-2022 public APIs
+
+    @property
+    def contributors(self) -> t.List[Contributor]:
+        '''
+        Returns a list of all the Contributors found in the metadata.
+        Returns [] if no Contributors exist.
+
+        >>> md = metadata.Metadata()
+        >>> md['composer'] = ['Richard Strauss']
+        >>> md.librettist = 'Oscar Wilde'
+        >>> md.add('title', 'Salome')
+        >>> contribs = md.contributors
+        >>> contribs
+        [<music21.metadata.primitives.Contributor composer:Richard Strauss>,
+        <music21.metadata.primitives.Contributor librettist:Oscar Wilde>]
+        '''
+        output: t.List[Contributor] = []
+        for namedValue in self.getAllContributorNamedValues():
+            assert isinstance(namedValue[1], Contributor)
+            output.append(namedValue[1])
+        return output
+
+    @property
+    def copyright(self):
+        '''
+        Returns the copyright as a str.
+        Returns None if no copyright exists in the metadata.
+        Returns all the copyright values in one string (with ', ' between them)
+        if multiple copyrights exist in the metadata.
+
+        Use md['copyright'] to get all the copyrights.
+
+        >>> md = metadata.Metadata()
+        >>> md.copyright is None
+        True
+        >>> md['dcterms:rights'] = ('Copyright © 1984 All Rights Reserved',)
+        >>> md.copyright
+        'Copyright © 1984 All Rights Reserved'
```

Comment:
show the attribute setter and getter version first.  Explain that dcterms:rights is the Dublin Core terminology for copyrights

Reply:
Done.